src/bindings/capi/thorvg_capi.h

```diff
@@ -950,15 +970,21 @@ TVG_API Tvg_Result tvg_paint_get_composite_method(const Tvg_Paint* paint, const
 /**
 * \brief Gets the unique id value of the paint instance indicating the instance type.
```

Comment:
maybe not to use 'id'? but up to you
`Gets the unique value indicating the type of the paint instance.`

Reply:
same with gradient